src/components/home/components/volt_sale_card.jsx

```diff
@@ -150,18 +154,19 @@ const VoltSaleCard = () => {
         <div className='grid-x grid-margin-x align-bottom'>
           <div className='cell small-24 medium-12'>
             <NumericalInput
-              label='FUSE'
+              label='Fuse'
               value={typedValue}
               onChange={setTypedValue}
               onMax={onMax}
               showMax
             />
           </div>
           <div className='cell small-24 medium-12'>
+            <p style={{ marginBottom: '5px', marginRight: '5px', textAlign: 'right' }}>Balance: {fuseBalance ? Number(fuseBalance).toFixed(4) : 0}</p>
```

Comment:
@RudeWolfCoding Please do not use inline styles!

Reply:
Changing inline styling into SASS